salesforce/modules/soap/records.bal

```diff
@@ -27,3 +27,68 @@ public type ConvertedLead record {
     string leadId;
     string opportunityId?;
 };
+
+# Represents lead to be converted.
+#
+# + accountId - ID of the Account into which the lead will be merged
+# + accountRecord - Account record to create
+# + bypassAccountDedupeCheck - Specifies whether to eliminate duplicate copies of Account data 
+# + bypassContactDedupeCheck - Specifies whether to eliminate duplicate copies of Contact data  
+# + contactId - ID of the Contact into which the lead will be merged   
+# + contactRecord - Contact record to create 
+# + convertedStatus - Valid LeadStatus value for a converted lead 
+# + doNotCreateOpportunity - Specifies whether to create an Opportunity during lead conversion(false) or not (true)
+# + leadId - ID of the Lead to convert
+# + opportunityId - The ID of an existing opportunity to relate to the lead. The opportunityId and opportunityName 
```

Comment:
Follow [this](https://ballerina.io/learn/style-guide/annotations-documentation-and-comments/) guide to add doc comments

Reply:
Fixed https://github.com/ballerina-platform/module-ballerinax-sfdc/commit/b1c997b55945a002a236eed5813068a4a9e2e61b